Sources/MongoSwift/MongoSwift.swift

```diff
@@ -1,17 +1,18 @@
 import libmongoc
 
-/// Returns a `String` indicating the version of the driver.
-public func version() -> String {
-    return "testing"
-}
-
 /// A utility class for libmongoc initialization and cleanup.
 final public class MongoSwift {
+    /// Returns a `String` indicating the version of the driver.
+    public static func version() -> String {
+        return "0.0.2"
```

Comment:
hm this looks very easy to forget to update 😄 Perhaps we should come up with some sort of release script that will ensure this is always correct

Reply:
Also, could we potentially just make this a floating global constant? I originally created this function as a total skeleton to just have _something_ in the file

Reply:
yeah, we could write something checking that it matches the Cocoapods file I guess? 

I think it's kind of nice to namespace it within the `MongoSwift` class, since we would probably name it something like "MongoSwiftVersionString" anyway. I don't have very strong feelings though.... if we do end up keeping it on the class though, I realized it should be a computed property rather than a function